libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -537,9 +543,19 @@ void AP_RCProtocol_CRSF::process_link_stats_frame(const void* data)
             // this is an approximation recommended by Remo from TBS
             _link_status.rssi = int16_t(roundf((1.0f - (rssi_dbm - 50.0f) / 70.0f) * 255.0f));
         }
-    }
-
-    _link_status.rf_mode = MIN(link->rf_mode, 7U);
+    }    
+
+    _link_status.rf_mode = MIN(link->rf_mode, ELRS_MAX_RFMD_MODES); // Cap to avoid memory spills in the conversion tables
```

Comment:
It would probably be better to define this as RFMode::RF_MODE_MAX_MODES - RFMode::CRSF_MAX_MODES

Reply:
Done! And I also removed the 4Hz setting from ELRS, as that seems to not be used at all in ELRS. Instead, I renamed that field to CRSF_RF_MAX_MODES. It's in [5b6ddcb](https://github.com/ArduPilot/ardupilot/pull/25844/commits/5b6ddcb3fab536f5001b69e2b87a2ad68350220d)